sbi/neural_nets/embedding_nets/transformer.py

```diff
@@ -628,91 +628,123 @@ def forward(self, pixel_values: torch.FloatTensor) -> torch.Tensor:
 
 
 class TransformerEmbedding(nn.Module):
```

Comment:
Would you be for adding a short explanatory class docstring here? 

e.g., for an SBI user working with time series or images but not so familiar with transformers, give a concise overview how they can use this class? e.g., what means "vit" (for images), what means "is_causal" (for time series). etc. not a tutorial, just a brief high-level explanation. Maybe even with a short code `Example` block.  

When we add this docstring here on the top class level then it will show up nicely in the Sphinx Documentation, e.g., like with the `EnsemblePosterior` here: https://sbi.readthedocs.io/en/latest/reference/_autosummary/sbi.inference.EnsemblePosterior.html#sbi.inference.EnsemblePosterior